src/pybind/mgr/cephadm/module.py

```diff
@@ -1488,7 +1488,8 @@ def __init__(self, placement_spec):
         # type: (orchestrator.PlacementSpec) -> None
         self.placement_spec = placement_spec
 
-    def place(self):
+    def place(self, host_pool, count=None):
```

Comment:
This function definition is only used for type checking. Thus only affects make check